lte/gateway/python/magma/enodebd/enodeb_status.py

```diff
@@ -336,7 +347,42 @@ def get_operational_states(
             deviceID=serial_id,
             value=serialized.encode('utf-8')
         )
+        configured_serial_ids.append(serial_id)
+        states.append(state)
+
+    # Get S1 connected eNBs
+    s1_states = get_enb_s1_connected_states(configured_serial_ids, mconfig)
+    for state in s1_states:
         states.append(state)
+
+    return states
+
+
+def get_enb_s1_connected_states(configured_serial_ids, mconfig) -> List[State]:
+    states = []
+    enb_s1_connected = get_all_enb_connected()
+    for enb_id in enb_s1_connected:
+        sn, enb = get_sn_enb_id_registered(mconfig, enb_id)
+        if enb and sn not in configured_serial_ids:
+            status = EnodebStatus(enodeb_configured=False,
+                                  gps_latitude='N/A',
+                                  gps_longitude='N/A',
+                                  enodeb_connected=True,
+                                  opstate_enabled=False,
+                                  rf_tx_on=False,
+                                  rf_tx_desired=False,
+                                  gps_connected=False,
+                                  ptp_connected=False,
+                                  mme_connected=True,
+                                  fsm_state='N/A',
+                                  ip_address=enb.ip_address)
```

Comment:
what's the point of returning the IP address that's configured via the API? this is a config value, not runtime state